src/internal.c

```diff
@@ -10347,8 +10917,9 @@ static int BuildCertHashes(WOLFSSL* ssl, Hashes* hashes)
     }
 #if ! defined( NO_OLD_TLS )
     else {
-        BuildMD5_CertVerify(ssl, hashes->md5);
-        BuildSHA_CertVerify(ssl, hashes->sha);
+        ret = BuildMD5_CertVerify(ssl, hashes->md5);
```

Comment:
See above (about line 10894).
wc_Md5GetHash() and wcShaGetHash() return errors now.